config/opal_setup_wrappers.m4

```diff
@@ -172,7 +172,33 @@ EOF
 
     AS_IF([test -n "$rpath_args"],
           [WRAPPER_RPATH_SUPPORT=rpath
-           AC_MSG_RESULT([yes ($rpath_args)])],
+           cat > $rpath_script <<EOF
```

Comment:
This whole new code block is a giant amount of replicated code -- it looks nearly identical to the block in earlier in `OPAL_SETUP_RPATH`.

This code block should likely be extracted into its own macro that both `OPAL_SETUP_RPATH` and `OPAL_SETUP_RUNPATH` can invoke.  Indeed, it should probably be parameterized so that we can use this one macro for all 3 cases: 1) `OPAL_SETUP_RPATH` C, 2) `OPAL_SETUP_RPATH` Fortran, and 3) `OPAL_SETUP_RUNPATH` Fortran.  That would make the code *much* more readable / maintainable.

I only mention this because it took me 15 minutes of closely reading to code to figure out why there were 3 copies of essentially the same Libtool-flag-extracting code (only *one* of them is for C, the other two are for Fortran, but that's only different by a single `--tag=FC` token in 2 of the 3 copies).